uber/templates/jobs/staffers.html

```diff
@@ -70,7 +70,9 @@
 <div style="margin-left:10px">
     [D]: <i>Staffers marked with a "D" are department heads.</i> <br/>
     [M]: <i>Staffers marked with a "M" are assigned to multiple locations</i> <br/>
-    [T]: <i>Staffers marked with a "T" are trusted</i> <br/>
+    [T]: <i>Staffers marked with a "T" are trusted
+         {% if location %}in this department{% else %}in at least one department{% endif %}
+         </i> <br/>
     [L]: <i>Staffers marked with a "L" are working load-in and/or load-out</i>
```

Comment:
@EliAndrewC ok I am happy with this report, let me know if this sounds acceptable to you. Pasted from the commit msg:

this is slightly tricky becuase there are 2 way to view this report 
1) viewing 'all' depts at once
2) viewing a specific department.

I did this:

if viewing all depts:
 - [T] will show on the report if this staffer is trusted in ANY dept

 if viewing a specific dept:
 - [T] will show on the report if this staffer is trusted in THAT dept

The text at the bottom changes as well.

This seems reasonable though if you weren't paying attention to the small text change, you might wonder why an attendee is Trusted in the big 'all' report, and then is not trusted in some specific department.  I can't think of a better solution here, except maybe on or more of:
1) just making that note about what 'trusted' means is bigger and bolder
2) in the 'all depts' view, replace [T] with something like [T(some depts)] or [T(1+)]